galaxy.yml

```diff
@@ -1,7 +1,7 @@
 ---
 namespace: community
 name: zabbix
-version: 2.0.0
+version: 2.0.1
```

Comment:
Please remove the change here.  We'll take care of it when we release an actual update.